File: AppGR/BackEnd/AppGrIT/Models/TokenModel.cs
using System.ComponentModel.DataAnnotations;

namespace BookManager.Model
{
    public class TokenModel
    {
        [Required]
        public string? AccessToken { get; set; }
        [Required]
        public string? RefreshToken { get; set; }
        [Required]
        public DateTime? Expiration { get; set;}
    }
}

File: AppChat/FontEnd/app/src/main/java/com/example/appchatit/models/TokenModel.java
package com.example.appchatit.models;

import java.time.LocalDateTime;

public class TokenModel {
    private String accessToken;
    private String refreshToken;
    private String expiration;

    public TokenModel() {
    }

    public TokenModel(String accessToken, String refreshToken, String expiration) {
        this.accessToken = accessToken;
        this.refreshToken = refreshToken;
        this.expiration = expiration;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }

    public void setRefreshToken(String refreshToken) {
        this.refreshToken = refreshToken;
    }

    public String getExpiration() {
        return expiration;
    }

    public void setExpiration(String expiration) {
        this.expiration = expiration;
    }
}